src/mixer/samplerbank.cpp

```diff
@@ -117,14 +136,22 @@ void SamplerBank::slotLoadSamplerBank(double v) {
         return;
     }
 
+    QString lastImportExportDirectory = m_pConfig->getValue(
+            kConfigkeyLastImportExportDirectory,
+            QStandardPaths::writableLocation(QStandardPaths::MusicLocation));
```

Comment:
Not sure if MusicLocation is the best fit for XML files, but I guess that is debatable and I don't really have a better suggestion.

Reply:
neither did I, that's why I picked it.
Though it is probably a good starting point, compared to having to navigate from the home folder to some other dir.

Reply:
this seems fine to me too